modules/apps/document-library/document-library-repository-search/build.gradle

```diff
@@ -1,9 +1,11 @@
+String luceneVersion = "8.6.2"
```

Comment:
@SamZiemer, better avoid this and change each version of the dependencies.

Other than that, looks good and we can forward after the tests pass :)